web/src/engine/websites/Anchira.ts

```diff
@@ -0,0 +1,95 @@
+import { Tags } from '../Tags';
+import icon from './Anchira.webp';
+import { type Chapter, DecoratableMangaScraper, Manga, Page, type MangaPlugin } from '../providers/MangaPlugin';
+import * as Common from './decorators/Common';
+import { FetchJSON } from '../platform/FetchProvider';
+import { Priority, TaskPool } from '../taskpool/TaskPool';
+import { RateLimit } from '../taskpool/RateLimit';
+
+type APIMangas = {
+    entries: APIManga[]
+}
+
+type APIManga = {
+    id: number,
+    key: string,
+    title: string
+    hash?: string,
+    names?: string[]
+}
+
+@Common.ChaptersUniqueFromManga()
+@Common.ImageAjax()
+export default class extends DecoratableMangaScraper {
+
+    private readonly mangasTaskPool = new TaskPool(4, new RateLimit(8, 1));
```

Comment:
The provided rate limit does not match the previous value, should be `2` requests per `1` second instead of `8` requests per `1` second.

Reply:
OK.